client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -17,16 +17,16 @@ const mergedHandlers = mergeHandlers(
 const shouldNext = action => {
 	const meta = action.meta;
 	if ( ! meta ) {
-		return true;
+		return false;
 	}
 
 	const data = meta.dataLayer;
 	if ( ! data ) {
-		return true;
+		return false;
```

Comment:
I've made those changes to make the tests pass once the `beenCalled` issue was detected, assuming that the tests were conceptually correct. We can always change the tests.